internal/cluster/clustersetup/velero/restore_backup_activity.go

```diff
@@ -44,6 +44,7 @@ const (
 var (
 	nonRestorableNamespaces = []string{
 		"kube-system",
+		"pipeline-system",
```

Comment:
**Question**: does this have any effect on the restore operation? (sorry, I don't have all the context information)

Reply:
Yes it affects, resources from pipeline-system are not restored when restoring to a new cluster. It doesn't make sense to restore these, since we install it anyway during cluster setup, it may cause problems in case you have integrated services like backup activated cause the restore process will install Velero from helm.